web3.js/bin/localnet.sh

```diff
@@ -84,9 +84,21 @@ up)
       --detach
       --name solana-localnet
       --rm
-      --publish 8899:8899
-      --publish 8900:8900
-      --publish 9900:9900
+      --publish 8001:8001/tcp # entrypoint
+      --publish 8001:8001/udp # gossip
+      --publish 8000:8000/udp # tvu
+      --publish 8002:8002/udp # tvu_forwards
+      --publish 8003:8003/udp # tpu
+      --publish 8004:8004/udp # tpu_forwards
+      --publish 8005:8005/udp # retransmit
+      --publish 8006:8006/udp # repair
+      --publish 8007:8007/udp # serve_repair
+      --publish 8008:8008/udp # broadcast
+      --publish 8899:8899 # rpc http
+      --publish 8900:8900 # rpc pubsub
+      --publish 8901:8901 # (future) bank service
+      --publish 8902:8902 # bank service
+      --publish 9900:9900 # faucet
```

Comment:
why not limit these to `/tcp` as well?

Reply:
The default is tcp only. No harm in being explicit though so I just updated these for clarity